collector/src/main/java/com/bitmovin/analytics/features/errordetails/ErrorDetail.kt

```diff
@@ -0,0 +1,5 @@
+package com.bitmovin.analytics.features.errordetails
+
+import com.bitmovin.analytics.features.segmenttracking.Segment
+
+data class ErrorDetail(val timestamp: Long, val code: Int?, val message: String?, val stacktrace: Array<String>?, val segments: Collection<Segment>?)
```

Comment:
why `Array<String>` ? why not listOf ? do we plan to change array items in runtime ?